### ISSUE TITLE
mwmVisit and mwmStar FITS don't work with cfitsio

### ISSUE THREAD

**Issue by ajwheeler** [sdss/astra#11]
This file is one of the most up-to-date, correct?  (It's 976 KB.)

```
julia> hdsu = FITS("mwmVisit-0.2.4dev-27021597919936514.fits")
File: mwmVisit-0.2.4dev-27021597919936514.fits
Mode: "r" (read-only)
HDUs: Num  Name        Type   
      1                Image  
      2    BOSS/APO    Table  
      3    BOSS/LCO    Table  
      4    APOGEE/APO  Table  
      5    APOGEE/LCO  Table  

julia> hdus[2] 
Error showing value of type TableHDU:
ERROR: CFITSIO has encountered an error. Error code 263: illegal TDIMn keyword value
Detailed error message follows: 
column vector length, 4648, does not equal TDIMn array size, 0
(4648, )
```

**Comment by andycasey** [comment]
It turns out this was because `TDIMn = (4648, )` instead of `TDIMn = (4648)`.

I've recreated this file and confirmed that I can read it with `fitsio`. It will be fixed in v0.2.5 for all the other mwmVisit/mwmStar files. You can find it at: https://data.sdss5.org/sas/sdsswork/mwm/spectro/astra/0.2.4dev/v6_0_9-daily/spectra/visit/6514/14/mwmVisit-0.2.4dev-27021597919936514.fits

Can you try read that updated file in julia?

**Comment by ajwheeler** [comment]
It works, thanks!
